x-pack/legacy/plugins/task_manager/lib/middleware.test.ts

```diff
@@ -145,29 +147,30 @@ Object {
       .beforeRun(getMockRunContext(getMockConcreteTaskInstance()))
       .then(contextOpts => {
         expect(contextOpts).toMatchInlineSnapshot(`
-Object {
-  "kbnServer": Object {},
-  "m1": true,
-  "m2": true,
-  "m3": true,
-  "taskInstance": Object {
-    "attempts": 0,
-    "id": "hy8o99o83",
-    "params": Object {
-      "abc": "def",
-    },
-    "primaryTerm": 1,
-    "retryAt": null,
-    "runAt": 2018-09-18T05:33:09.588Z,
-    "scheduledAt": 2018-09-18T05:33:09.588Z,
-    "sequenceNumber": 1,
-    "startedAt": null,
-    "state": Object {},
-    "status": "idle",
-    "taskType": "nice_task",
-  },
-}
-`);
+          Object {
```

Comment:
This might fail due to indentation woes. In other places I've seen us use `dedent` to remove excess whitespace (but preserve indentation like you have here). Might be worth using?

Reply:
This was the auto generated snapshot by Jest, so it should be failing seeing as it's how Jest is expecting it, no?